test/utils.js

```diff
@@ -22,7 +31,7 @@ module.exports.pause = function (seconds) {
 module.exports.copyRecursiveSync = function (origin, dest) {
   const stats = statSync(origin);
   if (stats.isDirectory()) {
-    mkdirp.sync(dest);
+    mkdirSync(dest, { recursive: true });
```

Comment:
is this working also with node 16 ?

Reply:
should be, iirc it got added in either node 14 or 16